build_bundle_image.sh

```diff
@@ -428,20 +428,20 @@ function push_docker_image {
 function set_parent_image {
 	if (is_quarterly_release "${LIFERAY_DOCKER_RELEASE_VERSION}")
 	then
-		if [[ "$(echo "${LIFERAY_DOCKER_RELEASE_VERSION}" | cut -d '.' -f 1)" -gt 2024 ]]
+		if [[ "$(get_release_year "${LIFERAY_DOCKER_RELEASE_VERSION}")" -gt 2024 ]]
 		then
 			return
 		fi
 
-		if [[ "$(echo "${LIFERAY_DOCKER_RELEASE_VERSION}" | cut -d '.' -f 1)" -eq 2024 ]] &&
+		if [[ "$(get_release_year "${LIFERAY_DOCKER_RELEASE_VERSION}")" -eq 2024 ]] &&
 		   [[ "$(echo "${LIFERAY_DOCKER_RELEASE_VERSION}" | cut -d '.' -f 2 | tr -d q)" -ge 3 ]]
```

Comment:
@davisaints, can you please create a new function in `release_common.sh` to get the quarterly release?